crates/dbsp/src/operator/dynamic/join.rs

```diff
@@ -422,6 +426,80 @@ impl<I1> Stream<RootCircuit, I1> {
     }
 }
 
+impl Stream<RootCircuit, MonoIndexedZSet> {
+    #[track_caller]
+    pub fn dyn_join_mono(
+        &self,
+        factories: &JoinFactories<MonoIndexedZSet, MonoIndexedZSet, (), OrdZSet<DynData>>,
+        other: &Stream<RootCircuit, MonoIndexedZSet>,
+        join_funcs: TraceJoinFuncs<DynData, DynData, DynData, DynData, DynUnit>,
+    ) -> Stream<RootCircuit, MonoZSet> {
+        self.dyn_join(factories, other, join_funcs)
+    }
+
+    #[track_caller]
+    pub fn dyn_join_index_mono(
+        &self,
+        factories: &JoinFactories<MonoIndexedZSet, MonoIndexedZSet, (), MonoIndexedZSet>,
+        other: &Stream<RootCircuit, MonoIndexedZSet>,
+        join_funcs: TraceJoinFuncs<DynData, DynData, DynData, DynData, DynData>,
+    ) -> Stream<RootCircuit, MonoIndexedZSet> {
+        self.dyn_join_generic(factories, other, join_funcs)
```

Comment:
is this right?

Reply:
yes, join_generic implements both join and join_index